newa/__init__.py

```diff
@@ -89,13 +91,24 @@ def render_template(
         raise Exception("Could not render template.") from exc
 
 
-def krb_get_request(url: str, attempts: int = 5, delay: int = 5) -> Any:
+def get_request(
+        url: str,
+        krb: bool = False,
+        attempts: int = 5,
+        delay: int = 5,
+        response_content: str = 'text') -> Any:
```

Comment:
It's better to use enums, linters can prevent you from making typos. With plain strings, nothing will prevent `get_request(..., response_content='tex')`.

```python
class ResponseContentType(enum.Enum):
  TEXT = 'text'
  JSON = 'json'
  ...

def get_request(..., response_content: ResponseContentType = ResponseContentType.TEXT)
  ...

  response = getattr(r, response_content.value)
```

No need for `if response_content in content_options` or `raise Exception` - mypy will prevent you from ever using a `response_content` value that's not in the enum.

Reply:
Yeah, I thought you will tell this. :-) Updated.